src/policy/permission.py

```diff
@@ -836,7 +836,19 @@ def perm_set_rse_usage(issuer, kwargs, session=None):
     :param session: The DB session to use
     :returns: True if account is allowed to call the API call, otherwise False
     """
-    return _is_root(issuer)
+    if _is_root(issuer):
+        return True
+
+    # Allow only site capacity managers to set usage
+    rse_id = kwargs.get('rse_id', None)
+    rse_attr = list_rse_attributes(rse_id=rse_id, session=session)
+    site_capacity_manager = rse_attr.get('site_capacity_manager', None)
+
+    if site_capacity_manager:
+        site_capacity_manager = site_capacity_manager.split(',')
```

Comment:
Please don't re-use a variable like this (changing from string to list). Just put a "s" on the end. Same comment at #872